Translation/Extractor/AbstractFileExtractor.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * This file is authored by PrestaShop SA and Contributors <contact@prestashop.com>
+ *
+ * It is distributed under MIT license.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\TranslationToolsBundle\Translation\Extractor;
+
+use Symfony\Component\Translation\Extractor\AbstractFileExtractor as BaseAbstractFileExtractor;
+use Symfony\Component\Translation\Extractor\ExtractorInterface;
+
+abstract class AbstractFileExtractor extends BaseAbstractFileExtractor implements ExtractorInterface
```

Comment:
Shouldn't this be put in `TraitExtractor` instead of creating an abstract class?